store/store.go

```diff
@@ -147,6 +148,14 @@ func (c *cacheTx) Commit() {
 	}
 }
 
+func (c *cacheTx) CommitNonce() {
+	for _, tx := range c.tmpTxs {
+		if tx.Action == txSet && util.HasPrefix(tx.Key, []byte("nonce")) {
```

Comment:
This is a hack, I don't want a hack, I want `NonceHandler` fixed.

Reply:
Alright, you got it.